### PR TITLE
Correctly detect cases that aren't found

### LINKED ISSUE #132: Searching for cases shows "You don't have permission"
It should tell user the case wasn't found.

### PR DESCRIPTION
**Pull request by orischwartz**
Closes #132

### CODE REVIEW

File: spec/features/download_spec.rb [hunk 1]
```diff
@@ -32,7 +32,7 @@
     expect(GetDownloadManifestJob).to have_received(:perform_later)
   end
 
-  scenario "Requesting non-existant case" do
+  scenario "Requesting non-existent case" do
```
Comment (shanear) [inline spec/features/download_spec.rb]: lol, sry

Comment (orischwartz) [inline spec/features/download_spec.rb]: That one was me actually

**Comment by shanear** [comment]
@orischwartz this needs a conflict merge :(

**Comment by shanear** [comment]
<3 it

**Pull request merged**